spring-mvc/src/main/java/com/kodemon/springmvc/controllers/UserController.java

```diff
@@ -244,13 +245,16 @@ public String reorder(ServletRequest r, Model model, Locale locale) {
         }
         UserDTO currentUser = (UserDTO) session.getAttribute("authenticatedUser");
         model.addAttribute("pokemons", currentUser.getPokemons());
+        model.addAttribute("activePokemons", currentUser.getActivePokemons());
+        model.addAttribute("numberOfPokemonsForFight", AMOUNT_OF_POKEMONS_FOR_MATCH);
         return "user/reorder";
     }
 
     @RequestMapping(value = "setFirstSixPokemons", method = RequestMethod.POST)
-    public String setFirstSixPokemons(ServletRequest r, Model model, Locale locale, @RequestParam int pokemon1,
-                                      @RequestParam int pokemon2, @RequestParam int pokemon3, @RequestParam int pokemon4,
-                                      @RequestParam int pokemon5, @RequestParam int pokemon6) {
+    public String setFirstSixPokemons(ServletRequest r, Model model, Locale locale, @RequestParam(required = false) Integer pokemon1,
```

Comment:
What if there will not be 6 active, but 12? :smile: Isn't there a way how to pass the list of integers here?